lib/matplotlib/_mathtext.py

```diff
@@ -1935,13 +1935,13 @@ def non_math(self, s, loc, toks):
     float_literal = staticmethod(pyparsing_common.convertToFloat)
 
     def _make_space(self, percentage):
-        # All spaces are relative to em width
+        # All spaces are relative to em width of the italic font style.
```

Comment:
May be worth noting that tex's spaces are the same regardless of whether you are in roman or italic, which is why we're doing the same here.  Perhaps also replace "em width" by "width of an "m"", as I believe (per the link I gave previously) that that's not really what an "em width" is...

Reply:
Done; I tried to concise while also getting all those ideas down.